### DISCUSSION TITLE
Creature balance/difficulty scaling issues

### DISCUSSION THREAD

**Discussion by Agent-Duck** [FakeFishGames/Barotrauma#9911] [Unstable feedback]
In my opinion monsters should be harder to deal with but attack less frequently, or at least maybe on a harder difficulty be this way, as presently not much of the aquatic fauna is actually that hard to deal with due to the addition of the new guns and the talent system, and simply making them appear more often at the harder difficulties doesn't really fix that.
If you have at least a pair of halfway competent gunners and an engineer you can deal with most anything on even the hardest difficulty with only minor hull damage
IMO this also makes the monsters feel less impactful, what used to be an "Oh crap a moloch" moment is now just "Oh look, a moloch, i'll get on guns after i've finished fixing this junction box", when there is so much of something it doesn't feel so special anymore or anywhere near as terrifying, even things like the endworm are more practically a matter of routine now.
This is especially the case with the addition of the new weapon size slot, as the new weapons & more powerful railgun look amazing, however if there are no balance changes to monsters they will seemingly make the game even less hard, as they are geared toward shredding monsters that generally already have a pretty hard time getting past AP coilguns.

TL;DR, The new guns look cool but please make enemies harder to compensate, at least on the higher difficulties, as simply having enemies attack more frequently or in greater numbers doesn't really increase difficulty at all, just makes the game sort of tedious as there is no downtime.

In previous versions of the game due to different balancing there were far greater periods of time between monster attacks, but due to the lack of offensive options they were much harder to deal with, this aided far more engaging gameplay as you were engaged in sustained periods of submarine maintainance/procedure punctuated by short bursts of high stress high stakes action, as opposed to at present an endless stream of mudraptors threshers and molochs which can be gunned down without breaking a sweat.

**Comment by gerakis** [comment]
I dunno man. Those mudraptors in the 3rd biome are getting ridiculous. They can cut into the ship almost instantly even with hull upgrades. Within 30 seconds we had 4 raptors aboard. They wrecked us so hard one guy even inquired how we made it this far in the game.

After failing that mission and restarting, we kited them and the gunners had more of a chance to deal with it, but wowee those things are strong.

**Reply by Agent-Duck** [reply]
I've never really seen that for a long while, from my experience even in the last stage of the campaign I haven't seen anything that can really remotely pose a threat to a well organised crew, mudraptors included, especially if you are using some of the new weapons like gatling guns.
I don't only think that the issue lies with the creatures themselves though, but the pacing, the fact that now you are basically constantly under attack and have no downtime, Which gets really tedious, and doesn't actually increase the difficulty.

Also I'd hardly describe 30 seconds as instantly, that's generally enough to kill off 4 mudraptors with even a coilgun, did you have someone spotting on sonar for you?

**Reply by LeakingAmps** [reply]
I'm in the first biome right now, but the other day I had some mudraptors go from outside sensor range to inside the sub in under 10 seconds. I had time to call them on radar, the guns got two of the 4, and then the other two didn't even slow down for the hull. Put a clip of revolver rounds in one and it just came back to eat my other arm. We've got half the hull upgrades, and are only on about 13% difficulty.

Mudrapters are honestly more dangerous than most of the large creatures right now.

That said, I do agree they should pace the attacks more; it's just tedious to constantly have something getting shot, after an hour or so.

**Reply by Ek-Videogames** [reply]
The small enemies are very much glass-cannons, they are allergic to bullets but if one sneezes in your general direction the submarine explodes.

Sonar's range & the unmaneuverability of the sub means - especially with some of the fastest enemies in the game like mudraptors & spinelings - you don't have much recourse to 'maneuver the sub' because by the time you've managed to cancel any forwards velocity they've likely already reached the hull, so in that regard it makes sense that they should be killable with guns before they touch the submarine.

Also agree with the statement that raptors are more dangerous than large creatures - because of their unavoidable speed.
I might also lump thresher-swarms in there because they are just as bad as mudraptors but have insane amounts of damage-resistance too. (and because their armor uses the 'old' method rather than mudraptor's detachable armor, threshers don't die instantly from anything AOE damage.)
